.github/workflows/ci.yaml

```diff
@@ -0,0 +1,15 @@
+on: [push, pull_request]
+jobs:
+  test:
+    name: Headless Tests
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    strategy:
      matrix:
        os: [ubuntu-latest, macos-latest]
    runs-on: ${{ matrix.os }}
```

Reply:
Done! :)

Wondering if adding `windows-latest` could bring any benefit.

Reply:
Thanks!

> Wondering if adding windows-latest could bring any benefit.

I have no plans to support Windows personally. So let's leave it for when we have an actual contributor who does use Windows and can help support it in case something breaks in the automated tests... But just not yet 😉